aptos-move/aptos-release-builder/data/example.yaml

```diff
@@ -9,7 +9,7 @@ proposals:
       discussion_url: ""
     execution_mode: SingleStep
     update_sequence:
-      - RawScript: data/proposals/empty.move
+      - RawScript: data/example_proposals/empty.move
```

Comment:
Make this relative to the repo

Reply:
Will have a separate PR then merge it into main first.